src/explore-education-statistics-admin/src/components/editable/__tests__/ReorderableAccordion.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import noop from 'lodash/noop';
+import Button from '@common/components/Button';
+import ReorderableAccordion from '../ReorderableAccordion';
+
+describe('ReorderableAccordion', () => {
+  test("it doesn't show controls if `canUpdateRelease` is false", () => {
+    const canUpdateRelease = false;
+
+    render(
+      <ReorderableAccordion
+        canUpdateRelease={canUpdateRelease}
+        id="test"
+        onReorder={() => noop}
+        heading="test"
+        onSectionOpen={() => noop}
+        onToggleAll={() => noop}
+      >
+        {canUpdateRelease && <Button>Reorder</Button>}
```

Comment:
Why try to render this? Given that `canUpdateRelease` is false, this would never render, but doesn't seem like we need this for the test at all

Reply:
Yep, not sure why I put this in. Think this is just a hangover from trying to rush this fix.

Reply:
I was getting a failure around no button being rendered since in the test it's rendering no children I assume

Reply:
If there are no children, we don't render the button so that should be another test case.

For the test cases on `canReorder`, it'd be better to try and render a more realistic test case e.g.

```tsx
<ReorderableAccordion canReorder>
  <ReorderableAccordionSection ...>
    Some test text
  </ReorderableAccordionSection>
</ReorderableAccordion>
```

Same applies for my other comment below.